### PR TITLE
Add Apache ActiveMQ Artemis catalog item

### PR DESCRIPTION
**Pull request by bostko**
Catalog item is visible in Apache Brooklyn karaf build
BROOKLYN-541

### CODE REVIEW

File: software/messaging/src/main/resources/catalog.bom [hunk 1]
```diff
@@ -25,6 +25,56 @@ brooklyn.catalog:
         type: org.apache.brooklyn.entity.messaging.activemq.ActiveMQBroker
         name: ActiveMQ Broker
         description: ActiveMQ is an open source message broker which fully implements the Java Message Service 1.1 (JMS)
+    - id: org.apache.brooklyn.entity.messaging.activemq-artemis
+      version: "1.0.0-SNAPSHOT" # BROOKLYN_VERSION
+      item:
+        type: org.apache.brooklyn.entity.software.base.VanillaSoftwareProcess
+        name: ActiveMQ Artemis
+        itemType: entity
+        description: Apache ActiveMQ Artemis has a proven non blocking architecture. It delivers outstanding performance. JMS 2.0 and 1.1 support.
```
Comment (michaelandrepearce) [inline software/messaging/src/main/resources/catalog.bom]: Might be worth a note that it is a message broker, maybe

"Apache ActiveMQ Artemis is an open source message broker and has a proven non blocking architecture. It delivers outstanding performance. JMS 2.0 and 1.1 support."

**Comment by michaelandrepearce** [comment]
@bostko thanks for this :)

File: software/messaging/src/main/resources/catalog.bom [hunk 2]
```diff
@@ -25,6 +25,56 @@ brooklyn.catalog:
         type: org.apache.brooklyn.entity.messaging.activemq.ActiveMQBroker
         name: ActiveMQ Broker
         description: ActiveMQ is an open source message broker which fully implements the Java Message Service 1.1 (JMS)
+    - id: org.apache.brooklyn.entity.messaging.activemq-artemis
+      version: "1.0.0-SNAPSHOT" # BROOKLYN_VERSION
+      item:
```
Comment (michaelandrepearce) [inline software/messaging/src/main/resources/catalog.bom]: be good to add an icon url

File: software/messaging/src/main/resources/catalog.bom [hunk 3]
```diff
@@ -25,6 +25,56 @@ brooklyn.catalog:
...
+        type: org.apache.brooklyn.entity.software.base.VanillaSoftwareProcess
```
Comment (michaelandrepearce) [inline software/messaging/src/main/resources/catalog.bom]: Im very new to brooklyn, whats the difference between this and a concrete type? It be good to have a concrete type? like the others?

**Comment by tbouron** [comment]
@bostko Have you seen @michaelandrepearce's suggestions. Do you think you can address them?

**Comment by bostko** [comment]
Thanks for the reminder @tbouron . Addressed comments.
Feel free to edit my PR if you have more comments.

**Comment by tbouron** [comment]
@bostko Sorry for the very late reply. It still miss an icon (see https://github.com/apache/brooklyn-library/pull/134#discussion_r146811001). After that, it should be good to go

**Comment by nakomis** [comment]
@tbouron Can you take a look at this please to check it's still valid and merge if you're now happy with it

**Comment by tbouron** [comment]
Thanks for the nudge @nakomis. LGTM, merging

**Pull request merged**